plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -170,7 +170,7 @@ public SqlServerClient(BaseJdbcConfig config, ConnectionFactory connectionFactor
     protected void renameTable(ConnectorSession session, String catalogName, String schemaName, String tableName, SchemaTableName newTable)
     {
         if (!schemaName.equals(newTable.getSchemaName())) {
-            throw new TrinoException(NOT_SUPPORTED, "Table rename across schemas is not supported");
+            throw new TrinoException(NOT_SUPPORTED, "This connector does not support renaming tables across schemas");
```

Comment:
:) Nice trick

Reply:
> :) Nice trick 

@nineinchnick @kokosing could you please explain what does it mean?



Could you please doublecheck if exception message corresponds to condition correctly.
I don't understand how case equality correlates with across schemas.